website/docs/r/catalog.html.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_catalog"
+sidebar_current: "docs-vcd-resource-catalog"
+description: |-
+Provides a vCloud Director catalog resource. This can be used to create and delete catalog.
+---
+
+# vcd\_catalog
+
+Provides a vCloud Director catalog resource. This can be used to create and delete catalog.
+
+Supported in provider *v2.0+*
+
+## Example Usage
+
+```
+resource "vcd_catalog" "myNewCatalog" {
+  org = "my-org"
+
+  name = "my-catalog"
+  description = "catalog for files"
+  delete_force      = "true"
+  delete_recursive  = "true"  
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to use. Not needed if defined at provider level
```

Comment:
Let's make it consistent with other files:
`The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations`